docker/docker-compose.yml

```diff
@@ -12,7 +12,17 @@ services:
       - "dbdata:/var/lib/postgresql/data"
   es:
     # Use the latest version of Elasticsearch supported by Amazon AWS ES.
-    image: "elasticsearch:6.7.2"
+    image: docker.elastic.co/elasticsearch/elasticsearch:6.7.2
+    ports:
+      - "9200:9200"
+      - "9300:9300"
+    environment:
+      - discovery.type=single-node
+      - bootstrap.memory_lock=true
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+    command: >
+      /bin/sh -c "./bin/elasticsearch-plugin list | grep -q analysis-icu 
+      || ./bin/elasticsearch-plugin install analysis-icu; tail -f /dev/null"
```

Comment:
Attn @nballenger, this was a necessary step for making real use of the `es` image.

Reply:
It's possible this command is enough by itself, if we just add a volume or two to persist the install. I pulled that ES image, and it looks like the entire install is contained under `/usr/share/elasticsearch`. If you set that as a volume it would keep the entire install across container runs. Alternatively you could add one or two volumes for `/usr/share/elasticsearch/plugins` and `/usr/share/elasticsearch/data`, if you wanted to just persist those bits.